oml/interfaces/models.py

```diff
@@ -17,4 +16,24 @@ def feat_dim(self) -> int:
         raise NotImplementedError()
 
 
+class IFreezable(ABC):
+    """
+    Models which can freeze and unfreeze their parts.
+    """
+
+    def freeze(self) -> None:
+        """
+        Function for freezing. You can use it to partially freeze a model.
+        Expected to be used with ``freeze_n_epochs`` in training config.
+        """
+        raise NotImplementedError()
+
+    def unfreeze(self) -> None:
+        """
+        Function for unfreezing. You can use it to unfreeze a model.
+        Expected to be used with ``freeze_n_epochs`` in training config.
```

Comment:
I think this interface does not need to know anything about config, so, let's remove
Expected to be used with ``freeze_n_epochs`` in training config.